java/src/jmri/jmrit/beantable/AbstractTableAction.java

```diff
@@ -139,6 +139,27 @@ public void dispose() {
         // should this also dispose of the frame f?
     }
 
+    /**
+     * Increments trailing digits of a system/user name (string)
+     * I.E. "Geo7" returns "Geo8"
+     * 
+     * Note: preserves leading zeros: "Geo007" returns "Geo008"
+     * Also, if no trailing digits, appends "1": "Geo" returns "Geo1"
+     * 
+     * @param name the system or user name string
+     * @return the same name with trailing digits incremented by one
+     */
+    protected @Nonnull String nextName(@Nonnull String name) {
+        final String[] parts = name.split("(?=\\d+$)", 2);
```

Comment:
Does this leave a (MERG i.e.) system name containing ``:`` alone, just incrementing the last part (called bit number?). Not familiar with the ``.split`` method.

Reply:
Will work to increment last number, seems fine. Mor ecomplex numbering schemes should not rely on "add range" and enter system names manually.